resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml

```diff
@@ -32,13 +32,27 @@ data:
       endpoints:
         - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
           honorLabels: true
-          port: http-metrics
-          scheme: http
+          port: https-metrics
```

Comment:
After that changes, what is the difference to the original kubelet servicemonitor?